server/plugins/index.js

```diff
@@ -31,7 +32,48 @@ function registerPlugins (server, config, callback) {
       }
     }
   })
-  var plugins = hapiPlugins.concat(localPlugins, hoodieCorePlugins)
+  if (!('plugins' in config)) config.plugins = []
```

Comment:
config.plugins should always be set per https://github.com/hoodiehq/hoodie/pull/482/files#diff-3cc7e17c48506403b02e5e3ae7089a2cR31